frontend/src/app/shared/components/datepicker/datepicker.modal.ts

```diff
@@ -209,6 +209,12 @@ export class DatePickerModalComponent extends OpModalComponent implements AfterV
   }
 
   cancel():void {
+    this.dates = {
+      date: '',
+      start: '',
+      end: '',
+    };
+    this.onDataChange();
```

Comment:
This will unset the dates, but the expected behavior is to reset the values to whatever is in the work package. I'm not sure the datepicker is the correct place to do that.

It might be easier to listen to the datepicker `closingEvent` and update it in the edit field